<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dolfinance</class>
 <widget class="QMainWindow" name="Dolfinance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dolfinance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="startDateInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2019</year>
      <month>1</month>
      <day>7</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDateInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2019</year>
      <month>6</month>
      <day>28</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Start date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>End date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stockTicker">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>GOOG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Stock ticker</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getData">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Get stock data</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="plotFrame">
     <header>PyQt5.QtWebEngineWidgets</header>
     <container>1</container>
     <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>920</width>
      <height>450</height>
     </rect>
    </property>
    </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
